cli/dts/lib.deno.ns.d.ts

```diff
@@ -250,6 +250,20 @@ declare namespace Deno {
   }
 
   export interface TestContext {
+    /**
+     * The full name of the current test.
+     */
+    name: string;
```

Comment:
I think this should probably just be the current test/step name. Then if someone wants them all they would be able to concatenate it themselves? Or perhaps we should have a `parent` property that gets the parent test context, then to get the full name someone could walk the parents.